controllers/helpdesk.js

```diff
@@ -7,6 +7,13 @@ const logger = require('winston');
 router.use(require('../helpers/authentication').authChecker);
 
 router.post('/', function (req, res, next) {
+    if (!req.body.subject && req.body.target) {
+        if(req.body.target === "HPI"){ // Contact Admin
```

Comment:
Ist HPI hier eine sinnvolle Bezeichnung? Für helpdesk gab es auch noch ein extra PR, nicht das es dort zu Konflikten kommt.

Reply:
Ich finde die auch schrecklich, aber ich habe nur den anderen PR per Reverse engineering hier eingebaut. Sollte im Phase 2 refactored werden, aber so funktioniert es jetzt. Kann leider nichts dafür, dass der Server so unspezifische Strings erwartet :/

Reply:
Nur fyi: Federführend in Sachen Helpdesk ist @Mupico - kein Blocker aber Lesbarkeit ist suboptimal.